app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,15 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="row.canDeploy()"
+                ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="!(row.canDeploy())"
+                ng-attr-aria-disabled="{{row.canDeploy() ? undefined : 'true'}}"
+                ng-class="{ 'disabled-link': !row.canDeploy() }">Deploy (paused)</a>
```

Comment:
OK I looked and there is no equivalent to "Deploy" that I can find for upstream deployments. `oc rollout latest` only works for deployment configs.